dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/AbstractMethodProblem.java

```diff
@@ -17,15 +17,22 @@
 package com.google.cloud.tools.opensource.classpath;
 
 /**
- * The referenced {@code methodSymbol} is not implemented in the {@code sourceClass} but the class
- * is declared to implement the method by {@code targetClass}. Such unimplemented methods manifest
- * as {@link AbstractMethodError}s at runtime.
+ * The referenced {@code methodSymbol} is not implemented in the {@code implementationClass} but the
```

Comment:
active voice: `{@code implementationClass} does not implement the abstract method {@code methodSymbol} declared by {@code interfaceOrAbstractClass}`

Reply:
Quick thought that I want to make sure you guys are thinking about. Presumably we still want to be able to add an exclusion to ignore an AbstractMethodProblem. The exclusions file uses the "source" and "target" terminology. Switching this to use implementationClass and interfaceOrAbstractClass might make it a bit confusion which one maps tot source/target when excluding. I personally thought calling the implementClass the "source" class was clear enough.